tests/virt_autotest/login_console.pm

```diff
@@ -95,10 +103,12 @@ sub login_to_console {
     use_ssh_serial_console;
 
     #workaround for bsc#1123942
```

Comment:
LGTM.

Reply:
You can ignore this workaround. Now the bug is fixed,  I will later open a separate PR to revert it. Even before revert, it will not affect any qemu/xen tests according to our job group which covers all hosts sle11sp4/12sp3/12sp4/15/s15sp1.